src/effects/effectslot.h

```diff
@@ -108,6 +109,8 @@ class EffectSlot : public QObject {
     QList<EffectParameterSlotPointer> m_parameters;
     QList<EffectButtonParameterSlotPointer> m_buttonParameters;
 
+    SoftTakeover* m_pSoftTakeover;
```

Comment:
Yes, though you can make it a member variable so you don't have to allocate it at all.